test/ringpop-assert.js

```diff
@@ -858,8 +851,23 @@ function validate(t, tc, scheme, deadline) {
 
     tc.on('event', function(event) {
         eventList.push(event);
+        allEvents.push(event);
```

Comment:
We need a second list, because events that are handled by the test are removed from the `eventList`.